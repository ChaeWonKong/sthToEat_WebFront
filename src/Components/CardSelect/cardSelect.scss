@import "../../Styles/mixins";

@include desktop {
  .wrapper {
    width: 300px;
  }
}

@include mobile {
  .wrapper {
    width: 80%;
  }
}
