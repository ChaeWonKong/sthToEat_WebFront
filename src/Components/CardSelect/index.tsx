import React from "react";

const CardSelect: React.FC = () => {
  return <div>CardSelect</div>;
};

export default CardSelect;
