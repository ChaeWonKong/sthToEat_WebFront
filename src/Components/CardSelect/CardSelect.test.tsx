import React from "react";
import { shallow } from "enzyme";
import CardSelect from "./index";

describe("CardSelect", () => {
  const cardSelect = shallow(<CardSelect />);

  it("renders correctly", () => {
    expect(cardSelect).toMatchSnapshot();
  });
});
