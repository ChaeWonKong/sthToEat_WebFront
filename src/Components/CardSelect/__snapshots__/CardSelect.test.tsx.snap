// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSelect renders correctly 1`] = `
<div>
  CardSelect
</div>
`;
